Telegram/README.md

```diff
@@ -0,0 +1,46 @@
+# Telegram Integration Prototype
+
+
+### Prerequisites
+
+- Follow the instructions on the [main README file](https://github.com/GoogleCloudPlatform/dialogflow-integrations#readme) in the root directory of this repository.
+- Create and add your [GCP service account key](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) to the current folder.
+- Set up a Telegram account through the app on your mobile device or on [Telegram Web](https://web.telegram.org/k/).
+- Set up a [bot account](https://core.telegram.org/bots#6-botfather).
+    - Copy the Token string provided and paste it into the veriable __TELEGRAM_TOKEN__ in __server.js__ file.
+- Uncomment and fill in the variables on the top of the __server.js__ file.
+- In your local terminal type and run "npm install @google-cloud/dialogflow-cx".
```

Comment:
Not sure if it's necessary to install the library on local machine. This seems only for the local test and not the required steps for the integration to work.